src/components/CustomerLogos.jsx

```diff
@@ -0,0 +1,181 @@
+import Tesla from "../image/Tesla.svg"
+import Twitch from "../image/Twitch.svg"
+import Intel from "../image/Intel.svg"
+import Shell from "../image/Shell.svg"
+import Netflix from "../image/Netflix.svg"
+import Nestle from "../image/Nestle.svg"  
+import Fedex from "../image/Fedex.svg"
+import Disney from "../image/Disney.svg"
+import BMW from "../image/BMW.svg"
+import CocaCola from "../image/CocaCola.svg"
+import TransparentButton from "../generalComponents/TransparentButton";
+import { Link } from 'react-router-dom';
+import NewSponsor from "../subpages/NewSponsor";
+
+const StyledCustomerLogos = styled.div`
+width: 1440px;
+height: 877px;
+border: 1px; 0px 96px 0px;
+gap: 64px;
+display: flex;
+background: #1F2A37;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+`
+const HeadingCustomerLogos = styled.div`
+width:672px;
+height: 161px;
+gap: 16px;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+`
+const ContainerCustomerLogos = styled.div`
+width: 1280px;
+heigh: 460px;
+gap: 32px;
+display: flex;
+flex-direction: column;
+justify-content: space-around;
+align-items: center;
+`
+const RowCustomerLogos = styled.div`
+width: 1280px;
+height: 132px;
+display: flex;
+flex-direction: row;
+justify-content: center;
+align-items: center;
+`
+const CardCustomerLogos = styled.div`
+width: 320px;
+heigh: 132px;
+border-radius: 8px;
+gap: 20px;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+`
+const LogoCustomerLogos = styled.svg`
+width: ${(props) => props.width};
+height: 48px;
+display: flex;
+justify-content: center;
+align-items: center;
+`
+const LogoTextContainer = styled.div`
+width: 142px;
+height:78px;
+gap:10px;
+display: flex;
+flex-direction: column;
+justify-content: center;
+align-items: center;
+`
+const TextCustomerLogos = styled.div`
+width: ${(props) => props.width||"142px"};
+height: ${(props) => props.height || "20px"};
+font-family: Inter;
+font-weight:400;
+font-size: 16px;
+line-height: 20px;
+color: #6B7280;
+display: flex;
+justify-content: center;
+align-items: center;
+`
+
+const CustomerLogos = () => {
+    return (
+        <StyledCustomerLogos>
+            <HeadingCustomerLogos>
+                <HeadingText width="672px" height="45px" description="Donors, Partners & Sponsors" display="flex" justifyContent="center" alignItems="center"></HeadingText>
+                <SupportingText width="672px" height="60px" description="Here at flowbite we focus on markets where technology, innovation, and capital can unlock long-term value." display="flex" justifyContent="center" alignItems="center"></SupportingText>
+                <LinkButton description="Become a sponsor" width="168px" height="24px"><NewSponsor/></LinkButton>
+            </HeadingCustomerLogos>
+            <ContainerCustomerLogos>
+                <RowCustomerLogos>
+                    <CardCustomerLogos>
```

Comment:
Szereg powtarzalnych komponentów, należałoby stworzyć tablicę z danymi do wyświetlenia i ją tutaj mapować. tutaj bardziej to przypomina tradycyjny html